jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/exceptionmappers/BaseExceptionMapperSpec.scala

```diff
@@ -0,0 +1,31 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers
+
+import com.cerner.beadledom.jaxrs.models.JsonError
+
+import com.google.gson.Gson
+
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.{BeforeAndAfterEach, FunSpec, ShouldMatchers}
+
+import play.api.libs.json.{JsValue, Json}
+
+import javax.ws.rs.core.Response.Status
+
+/**
+  * Base spec to be used by ExceptionMapper spec classes.
+  *
+  * @author Cal Fisher
+  */
+class BaseExceptionMapperSpec
+    extends FunSpec with ShouldMatchers with BeforeAndAfterEach with MockitoSugar {
```

Comment:
Use `MustMatchers` instead of `ShouldMatchers.

Reply:
We tend to prefer `BeforeAndAfter` over `BeforeAndAfterEach` as well. The main disadvantage of `BeforeAndAfter` is that you can't write stackable traits using it (basically a trait that extends `BeforeAndAfter` and calls `super.before`), but we haven't had the need to do that (yet at least). The main win is that it offers a nicer synxax:

```scala
before {
  . . .
}
```
vs.

```scala
override def beforeEach: Unit = {
  . . .
}
```

Reply:
Updated to use `MustMatchers` and `BeforeAndAfter`: https://github.com/cerner/beadledom/pull/45/commits/b19a43a19a67ff8b1aa1723f780453e7bbeb55ac#diff-f4aa1ecb4a6061bdc29737b015c5e9e6R20